cfg/plus4-hires.cfg

```diff
@@ -0,0 +1,57 @@
+# Linker configuration that allows for a hi-res bitmap at $C000-$DF3F, but
+# puts the stack (and a "HIBSS" segment) in the remaining RAM at $DF40-$FD00.
+
+FEATURES {
+    STARTADDRESS: default = $1001;
+}
+SYMBOLS {
+    __LOADADDR__:  type = import;
+    __EXEHDR__:    type = import;
+    __STACKSIZE__: type = weak, value = $0800; # 2k stack
+    __HIMEM__:     type = weak, value = $FD00;
+    __RESERVED_C000__: type = weak, value = 8000; # Reserve 8000 bytes for hi-res bitmap
```

Comment:
Please elaborate why this is a symbol. Is there a valid use case where its value isn't 8000?

Reply:
The only reason is because I wanted to make it as general as possible, following the example of `__RESERVED_MEMORY__` in atari.cfg. Happy to remove it if it's not wanted.

Reply:
Afaik there are quite some graphics modes and setups to be supported by the Atari cfg. I'd suggest to just go with plain constants here - at least as long as there's only one TGI driver to support.